packages/core/src/components/Notification/Wrappers/NotificationDrawerPanelWrapper.js

```diff
@@ -160,6 +175,7 @@ NotificationDrawerPanelWrapper.defaultProps = {
   className: null,
   panelName: null,
   onNotificationClick: null,
+  onNotificationAsRead: null,
```

Comment:
The two callbacks should be defaulted to noop, then no need to check if non-null above.